hedera-node/src/main/java/com/hedera/services/store/models/NftId.java

```diff
@@ -45,4 +48,14 @@ public static NftId fromGrpc(final TokenID tokenId, final long serialNo) {
 	public EntityNumPair asEntityNumPair() {
 		return EntityNumPair.fromLongs(num, serialNo);
 	}
+
+	@Override
+	public int compareTo(final @NotNull NftId that) {
+		return NATURAL_ORDER.compare(this, that);
+	}
+
+	private static final Comparator<NftId> NATURAL_ORDER = Comparator.comparingLong(NftId::num)
```

Comment:
Can we move the static members to top of class ? If we're moving to google java style, this should be at top of class.